Bio/Cluster/__init__.py

```diff
@@ -42,10 +42,16 @@
 
 
 class Node(_cluster.Node):
+    """Define an element of a hierarchical clustering tree.
```

Comment:
Why "Define"? I would think just "Element of a hierarchical clustering tree." is better here.

Reply:
Agree